reconcile/user_validator.py

```diff
@@ -26,12 +27,13 @@ def validate_users_single_path(users):
     return ok
 
 
-def validate_users_gpg_key(users):
+def validate_users_gpg_key(users, smtp_client):
     ok = True
     for user in users:
         public_gpg_key = user.get('public_gpg_key')
         if public_gpg_key:
-            gpg_ok, error_message = gpg_key_valid(public_gpg_key)
+            recepient = smtp_client.get_recipient(user['org_username'])
+            gpg_ok, error_message = gpg_key_valid(public_gpg_key, recepient)
```

Comment:
The interface for `gpg_key_valid` kills me. In many other places we return the `ok` flag as the last element of the result tuple. And, if the next caller makes that assumption and does:

```python
message, ok = gpg_key_valid(public_gpg_key, recipient)
```
there is nothing to detect the mistake. No linter, no crashing until much farther in the code... just wrong results, and most tests wouldn't even catch it.